packages/manager/core/application/src/ovh-routing.tsx

```diff
@@ -36,6 +45,16 @@ function HidePreloader(): JSX.Element {
 
 function OvhContainerRoutingSync(): JSX.Element {
   const location = useLocation();
+  const outlet = useOutlet();
+  const navigate = useNavigate();
+  useEffect(() => {
+    setTimeout(() => {
+      const child = outlet.props?.children?.props?.children?.props?.children;
```

Comment:
**question:** Why is this nesting required?
Also, this seems like a bad approach as `props.childern` can have multiple levels of children